products/dataproc/api.yaml

```diff
@@ -219,6 +220,19 @@ objects:
         input: true
         description: |
           The region in which the cluster and associated nodes will be created in.
+      - !ruby/object:Api::Type::String
+        name: 'gracefulDecommissionTimeout'
+        url_param_only: true
+        input: true
```

Comment:
Is this not updatable?  the addition of an update_url makes me think it is, but input: true makes me think it isn't.

Reply:
Oh, I see - this isn't getting used, really, this resource is handwritten.  You can probably revert this change if you like.

Reply:
cool, will do ! :)